libs/sdk-ui-web-components/src/tests/context.test.ts

```diff
@@ -0,0 +1,33 @@
+// (C) 2022 GoodData Corporation
+import { CustomElementContext } from "../context";
+
+describe("context", () => {
+    // Use enforced type casting for test simplicity
+    const mockContext = { backend: "test", workspaceId: "test" } as unknown as CustomElementContext;
```

Comment:
You could avoid the cast by providing a "real" backend, see https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-charts/src/charts/areaChart/tests/AreaChartStacking.test.ts#L6 for example.